NotificationService/NotificationService.swift

```diff
@@ -0,0 +1,35 @@
+//
```

Comment:
This looks like its duped with `OptimoveNotificationService`

Reply:
Suggest we just roll with `NotificationService` you already have the Optimove SDK, we probably dont need to qualify it again?